Packs/AzureStorageContainer/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Azure Storage Container",
     "description": "Create and Manage Azure Storage Container services.",
     "support": "xsoar",
-    "currentVersion": "1.0.22",
+    "currentVersion": "2.0.0",
```

Comment:
We should leave it with 1.0.23. Please run the `demisto-sdk update-release-notes` command again with the `-u revision` argument